### PR TITLE
88 teaser cards

### LINKED ISSUE #88: Teaser cards
[Visual adjustment](https://www.figma.com/file/yv0VtHTvMoGPqkv0RAYcVp/PDP?type=design&node-id=6610-7770&mode=dev) of existing teaser cards. We will create a new block with v2- prefix. Related to #11; refactoring required.

### Requirements
- Style changes: sizing, typography, colours, background, rounded corners
- ~~On mobile, the cards will be scrollable~~
- When the amount of cards is odd, the last item will be as wide as the container on tablet/desktop
- Colours change [according to context](https://www.figma.com/file/Id3nd9kbfh8TpS1SfWhrW2/Volvo-Component-Library?type=design&node-id=398-8212&mode=design&t=ynTTAjqhDm7J7Zly-4)

### PR DESCRIPTION
**Pull request by cogniSyb**
Fix #88

Test URLs:
- Before: https://main--vg-volvotrucks-us-rd--netcentric.hlx.page/drafts/syb/teaser-cards
- After: https://88-teaser-cards--vg-volvotrucks-us-rd--netcentric.hlx.page/drafts/syb/teaser-cards

Based on #11 and #13; will replace the former. The block v2-images-grid can be replaced. I haven’t removed those files yet, so that the code/outcome can be compared.

Some global styles have been adjusted, regarding
- wrapper-width
- tertiary button
- pretitle

### Known issues
- [x] Button size/alignment is off (hover/active state)
- [x] Button size (length) should be controlled better
- [x] Some styles regarding text/button colour in cards could be made global
- [x] Accessibility issue with video not being focusable (just as with #13)

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -1263,19 +1264,25 @@ main.blue-contract .section.section-with-title p {
 
 .redesign-v2 .pretitle {
   font-feature-settings: 'clig' off, 'liga' off;
-  font-size: var(--f-subtitle-2-font-size);
-  letter-spacing: var(--f-subtitle-2-letter-spacing);
-  line-height: var(--f-subtitle-2-line-height);
+  font-size: var(--f-body-2-font-size);
```
Comment (kesiah) [inline styles/styles.css:1267]: This change is going to break down: https://github.com/Netcentric/vg-volvotrucks-us-rd/pull/83
I agree that all pretitles should look the same, but which one is the one that we should follow?

**Review by kesiah — CHANGES_REQUESTED** [review]
Hover tertiary button looks weird:
image

In tablet view, the padding is not the same as the header, one of them is wrong.
Screenshot 2023-09-19 at 09 02 11

**Review by Lakshmishri — CHANGES_REQUESTED** [review]
Screenshot 2023-09-19 at 12 07 15
There should be some spacing between the image and button on hover.

General question:
Is this how the tertiary button would look on hover?

**Review by kesiah — APPROVED** [review]
As the buttons are going to be solved in a separate ticket, I approve!

**Review by Lakshmishri — APPROVED** [review]

**Pull request merged**